toggl/translations
# Remove package build files from repo

The idea was to do that via GPR, but Github Actions still doesn't support reading from Github Packages so I went for simpler steps:
1. Remove the files
2. Alter workflow to build before executing the action

To test the workflow:
```
curl -u <your_github_email>:<personal_token> \
  -X POST \
  -H "Accept: application/vnd.github.v3+json" \
  https://api.github.com/repos/toggl/translations/actions/workflows/main.yml/dispatches \
  -d '{"ref":"publish-action-workflow"}'
```

☝️ Your personal token must have `repo` permissions, you can generate one here https://github.com/settings/tokens

Check if your workflow did run and it's 🟢 ✨ https://github.com/toggl/translations/actions

part of toggl/webapp#14179
